src/Compilers/Core/MSBuildTaskTests/TestUtilities/IntegrationTestBase.cs

```diff
@@ -199,20 +199,21 @@ public void StdLib_Csc(bool useSharedCompilation, bool disableSdkPath)
             new Dictionary<string, string>
             {
                 { "File.cs", """
+                    using System.Linq;
                     System.Console.WriteLine("Hello from file");
                     """ },
                 { "Test.csproj", $"""
                     <Project>
                         <UsingTask TaskName="Microsoft.CodeAnalysis.BuildTasks.Csc" AssemblyFile="{_buildTaskDll}" />
                         <Target Name="CustomTarget">
-                            <Csc Sources="File.cs" UseSharedCompilation="{useSharedCompilation}" DisableSdkPath="{disableSdkPath}" />
+                            <Csc Sources="File.cs" UseSharedCompilation="{useSharedCompilation}" DisableSdkPath="{disableSdkPath}" NoConfig="{noConfig}" />
```

Comment:
>PairwiseData [](http://example.com/codeflow?start=13&length=12)

What does this do and why the change is necessary?

Reply:
It avoids exponential amount of test runs since I added new input. See https://aarnott.github.io/Xunit.Combinatorial/docs/combinatorial-vs-pairwise.html. The pairwise ones should be the interesting ones (I've verified the test still fails without the product change for example).

Reply:
I do not find 8 cases too much and would be more comfortable with covering full matrix. I am uncomfortable with a situation when an "external" tool decides what combinations our tests cover and that combinations possibly changing between versions of that tool. My recommendation is to use full matrix or explicitly specifying combinations covered by the test.